### ISSUE TITLE
SteamOS 3.4.4 Beta (2/17 Release): Clicking Badge No Longer Opens Up ProtonDB Page

### ISSUE THREAD

**Issue by wonderwallwoody** [OMGDuke/protondb-decky#48]
With the most recent SteamOS Beta release (on 2/17), the ProtonDB badges do not open up the corresponding protondb.com page. This seems to be due to the issue brought up here: https://github.com/SteamDeckHomebrew/decky-loader/issues/371#issuecomment-1435484203

From the CEF Debugger, here's the error when trying to click the button:
```
Uncaught (in promise) TypeError: Router.NavigateToExternalWeb is not a function
    at onClick (eval at importReactPlugin (index.js:1866:43), <anonymous>:3669:29)
    at Object.p (libraries~00299a408.js:2:803223)
    at u (libraries~00299a408.js:2:803366)
    at libraries~00299a408.js:2:803512
    at f (libraries~00299a408.js:2:803598)
    at pt (libraries~00299a408.js:2:819114)
    at it (libraries~00299a408.js:2:818924)
    at st (libraries~00299a408.js:2:819280)
    at lt (libraries~00299a408.js:2:820487)
    at C (libraries~00299a408.js:2:917250)
```

**Comment by wonderwallwoody** [comment]
Bumping the DFL version should work (as per https://discord.com/channels/960281551428522045/960284327445418044/1076320523539189770)

**Comment by OMGDuke** [comment]
A fix for this has bit submitted as v1.0.7. It's available manually from the releases page of this repo.

It should be on the testing version of the decky plugin store shortly!

**Comment by OMGDuke** [comment]
Thanks for the report!
